pkg/i2gw/providers/gce/storage.go

```diff
@@ -17,16 +17,26 @@ limitations under the License.
 package gce
 
 import (
+	apiv1 "k8s.io/api/core/v1"
 	networkingv1 "k8s.io/api/networking/v1"
 	"k8s.io/apimachinery/pkg/types"
+	backendconfigv1 "k8s.io/ingress-gce/pkg/apis/backendconfig/v1"
 )
 
 type storage struct {
 	Ingresses map[types.NamespacedName]*networkingv1.Ingress
+
+	// BackendConfig is a GKE Ingress extension, and it is associated to an GKE
+	// Ingress through specifying `cloud.google.com/backend-config` or
+	// `beta.cloud.google.com/backend-config` annotation on its Backend Services.
```

Comment:
A few nits here:
1. This comment feels like it belongs directly above BackendConfigs line, not Services.
2. Recommend mentioning that BackendConfig map is keyed by _Service_ namespace/name, not BackendConfig namespace/name. Or I'm wrong here, either way, would be good to clarify.
3. Recommend just saying "Services" instead of "Backend Services".

Reply:
Updated, thanks for the suggestion!